test/integration/guest_can_create_account_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.